examples/DictionaryExample/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Momento.Sdk.Exceptions;
+using Momento.Sdk.Incubating;
+
+public class Driver
+{
+    private static readonly string AUTH_TOKEN_ENV_VAR = "TEST_AUTH_TOKEN";
+    private static readonly string CACHE_NAME_ENV_VAR = "TEST_CACHE_NAME";
+
+    public async static Task Main()
+    {
+        var authToken = ReadAuthToken();
+        var cacheName = ReadCacheName();
+
+        using var client = Momento.Sdk.Incubating.SimpleCacheClientFactory.CreateClient(authToken, 60);
+        EnsureCacheExists(client, cacheName);
+
+        // Set a value
+        await client.DictionarySetAsync(cacheName: cacheName, dictionaryName: "my-dictionary",
+            field: "my-field", value: "my-value", refreshTtl: false, ttlSeconds: 60);
+
+        // Set multiple values
+        await client.DictionarySetBatchAsync(
+            cacheName: cacheName,
+            dictionaryName: "my-dictionary",
+            new Dictionary<string, string>() {
+                { "field1", "value1" },
+                { "field2", "value2" },
+                { "field3", "value3" }},
+            refreshTtl: false);
+
+        // Get a value
+        var field = "field1";
+        var getResponse = await client.DictionaryGetAsync(
+            cacheName: cacheName,
+            dictionaryName: "my-dictionary",
+            field: field);
+        var status = getResponse.Status; // HIT
+        string value = getResponse.String()!; // "value1"
```

Comment:
This stuff is going to change in Erika's PR, right?  Is the goal here to give something to Tom that he can use a reference before we switch over?

Reply:
Discussed on slack